### ISSUE TITLE
Reset curbid to ensure demand is not used multiple times

### ISSUE THREAD

**Issue by ix-certification** [indexexchange/kargo-adapter#5]
We would suggest reset the `curBid` being defined here
https://github.com/indexexchange/kargo-adapter/blob/c1f78bf11e5f0304662306e92eb692ca7b1eb6e0/kargo-htb.js#L364
to `null` after its been consumed. One potential place to add it would be before this block
https://github.com/indexexchange/kargo-adapter/blob/c1f78bf11e5f0304662306e92eb692ca7b1eb6e0/kargo-htb.js#L425

**Comment by samuelhorwitz** [comment]
I'm not familiar with this new code. It looks like my code was refactored 6 days ago by the Index team. This issue seems to be related to code present only in the refactor, specifically boilerplate code as opposed to adapter logic code. While I can submit a PR adding the one line fix as noted, I am a bit confused as to whether I am understanding the problem correctly.

**Comment by jas-singh** [comment]
The older adapter that was written by the Kargo team was updated by IX in order to make sure its compatible with the fall release of the wrapper which provides features such as Safeframe support, out of page support. After successfully completing this update, we are now at a point where we can see that the management of the adapter can be owned by your team. In addition to the features added in fall release, we've also updated the adapter documentation and a unit testing suite which should give you & your team a good starting point to get acquainted with the adapter code.

**Comment by ix-certification** [comment]
The older adapter that was written by the Kargo team was updated by IX in order to make sure its compatible with the fall release of the wrapper which provides features such as Safeframe support, out of page support. After successfully completing this update, we are now at a point where we can see that the management of the adapter can be owned by your team. In addition to the features added in fall release, we've also updated the adapter documentation and unit testing suite which should give you & your team a good starting point to get acquainted with the adapter code.
